lib/texture.js

```diff
@@ -953,14 +953,17 @@ module.exports = function createTextureSet (
     // but we do not yet support having multiple mipmap levels for compressed textures,
     // so we only test for level zero.
 
-    if (mipmap.compressed &&
-        (mipmap.internalformat === GL_COMPRESSED_RGB_S3TC_DXT1_EXT) ||
-        (mipmap.internalformat === GL_COMPRESSED_RGBA_S3TC_DXT1_EXT) ||
-        (mipmap.internalformat === GL_COMPRESSED_RGBA_S3TC_DXT3_EXT) ||
-        (mipmap.internalformat === GL_COMPRESSED_RGBA_S3TC_DXT5_EXT)) {
-      check(mipmap.width % 4 === 0 &&
-            mipmap.height % 4 === 0,
-            'for compressed texture formats, mipmap level 0 must have width and height that are a multiple of 4')
+    if (
+      mipmap.compressed &&
+      (
+        mipmap.internalformat === GL_COMPRESSED_RGB_S3TC_DXT1_EXT ||
+        mipmap.internalformat === GL_COMPRESSED_RGBA_S3TC_DXT1_EXT ||
+        mipmap.internalformat === GL_COMPRESSED_RGBA_S3TC_DXT3_EXT ||
+        mipmap.internalformat === GL_COMPRESSED_RGBA_S3TC_DXT5_EXT
+      )
```

Comment:
Here I made a semantic change (maybe bugfix?) - see that in the original code, the `mipmap.compressed` only applied to the 1st element of the `||` chain - from the comment and indentation it looks like maybe my version was intended